src/poliastro/threebody/soi.py

```diff
@@ -12,7 +12,7 @@
 The result is:
 
 .. math::
-    a\\times(\\frac{m}{M}^{\\frac{2}{5}})
+    a\\times(\\frac{m}{M})^{\\frac{2}{5}}
```

Comment:
While we are at it, can we remove `\times` and use `\left(` and `\right)` for the delimiters so the correct size is chosen?

Reply:
Current appearance: